index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta char="utf-8">
+	<title>F to C | C to F</title>
+	<link rel="stylesheet" type="text/css" href="../css/main.css">
+</head>
+<!-- In the HTML, have one input field where someone can enter in a temperature. -->
+	<body>
+		<form>
+			<fieldset>
+				<legend>Convert temperatures here!</legend>
+				<label>Temperature you'd like to convert:</label>
+				<input id="input-text" type="text" placeholder="Type Here">
+
+<!-- Create a radio button group where Celsius or Fahrenheit can be selected as the scale that the number should be converted to. -->
+				<input type="radio" name="fahrenheit" id="fahrenheit">Fahrenheit
+				<input type="radio" name="celsius" id="celsius">Celsius	
```

Comment:
Here, for both of these radio buttons, change the "name" attribute to "temp" (or something similar) on both.  This will group them and prevent the user from selecting both at once.